lib/overcommit/hook/pre_push/git_lfs.rb

```diff
@@ -0,0 +1,21 @@
+module Overcommit::Hook::PrePush
+  # Invokes Git LFS command that uploads files tracked by Git LFS to the LFS storage
+  #
+  # @see https://git-lfs.github.com/
+  class GitLfs < Base
+    def run
+      result = execute(['command', '-v', 'git-lfs'])
+      unless result.success?
+        return :warn, 'This repository is configured for Git LFS but \'git-lfs\' ' \
+        'was not found on your path.\nIf you no longer wish to use Git LFS, ' \
+        'disable this hook by removing or setting \'enabled: false\' for GitLFS ' \
+        'hook in your .overcommit.yml file'
+      end
+
+      result = execute(['git', 'lfs', 'pre-push', remote_name, remote_url])
+      return :fail, result.stderr unless result.success?
+
+      :pass
```

Comment:
This would require some refactoring of the display logic for Overcommit. In the past we've kicked around the idea of supporting progress bars or similar, but this would require a much more complicated display controller that utilized more advanced cursor features so you could update multiple lines at the same time (since hooks run in parallel).